datadog_checks_dev/datadog_checks/dev/tooling/commands/release/show/changes.py

```diff
@@ -49,37 +58,32 @@ def changes(ctx, check, tag_pattern, tag_prefix, dry_run, organization, since, e
     if not dry_run:
         echo_info(f'Found {len(pr_numbers)} PRs merged since tag: {target_tag}')
 
+    applicable_changelog_types = []
     user_config = ctx.obj
-    if dry_run:
-        changelog_types = []
-
-        for pr_num in pr_numbers:
-            try:
-                payload = get_pr(pr_num, user_config, org=organization)
-            except Exception as e:
-                echo_failure(f'Unable to fetch info for PR #{pr_num}: {e}')
-                continue
-            current_changelog_types = get_changelog_types(payload)
-            if not current_changelog_types:
+    echo_debug(f'Evaluating PRs: {pr_numbers}')
+    for pr_num in pr_numbers:
+        try:
+            payload = get_pr(pr_num, user_config, org=organization)
+        except Exception as e:
+            echo_failure(f'Unable to fetch info for PR #{pr_num}: {e}')
+            continue
+        changelog_types = get_changelog_types(payload)
+        if dry_run:
+            echo_debug('Running changes in dry run mode. Command will abort if there are invalid tags')
+            if not changelog_types:
+                echo_failure(f'No valid changelog labels found attached to PR #{pr_num}, please add one!')
                 abort(f'No valid changelog labels found attached to PR #{pr_num}, please add one!')
-            elif len(current_changelog_types) > 1:
+            elif len(changelog_types) > 1:
+                echo_failure(f'Multiple changelog labels found attached to PR #{pr_num}, please only use one!')
                 abort(f'Multiple changelog labels found attached to PR #{pr_num}, please only use one!')
 
-            current_changelog_type = current_changelog_types[0]
+            current_changelog_type = changelog_types[0]
             if current_changelog_type != 'no-changelog':
-                changelog_types.append(current_changelog_type)
-
-        return cur_version, changelog_types
-    else:
-        for pr_num in pr_numbers:
-            try:
-                payload = get_pr(pr_num, user_config, org=organization)
-            except Exception as e:
-                echo_failure(f'Unable to fetch info for PR #{pr_num}: {e}')
-                continue
-
-            changelog_types = get_changelog_types(payload)
-
+                echo_debug(f'Found change {current_changelog_type}')
+                applicable_changelog_types.append(current_changelog_type)
+            else:
+                echo_debug(f'Found no-change change for PR {pr_num}, skipping')
```

Comment:
```suggestion
                echo_debug(f'Found no-changelog change for PR {pr_num}, skipping')
```
`no-change change` is a bit confusing